pages/common/dnsx.md

```diff
@@ -0,0 +1,37 @@
+# dnsx
+
+> A fast and multi-purpose DNS toolkit allow to run multiple DNS queries.
+> Note: input to `dnsx` needs to be passed through `stdin` (pipe `|`) in some cases.
+> More information: <https://github.com/projectdiscovery/dnsx>.
+
+- Query the A record of a (sub)domain:
+
+`echo {{example.com}} | dnsx -a -re`
+
+- Query all the dns records (A,AAAA,CNAME,NS,TXT,SRV,PTR,MX,SOA,AXFR,CAA):
+
+`dnsx -recon -re <<< {{example.com}}`
+
+- Query a specific type of DNS record:
+
+`echo {{example.com}} | dnsx -re -{{a|aaaa|cname|ns|txt|srv|ptr|mx|soa|any|axfr|caa}}`
+
+- Output [r]esponse [o]nly (do not show the queried domain or subdomain):
+
+`echo {{example.com}} | dnsx -ro`
+
+- Display raw response of a query (`dig` style output), specifying [r]esolvers to use and retry attempts for failures:
```

Comment:
```suggestion
- Specify output style, [r]esolvers and number of retry attempts for failures:
```

Reply:
The option is not really meant to display different styles of output. I added the `dig style output` as a hint.